makemkv/makemkv.py

```diff
@@ -283,7 +282,7 @@ def _run(self, cmd: list[str]) -> MakeMKVOutput:
                 loglevel = MESSAGE_CODES.get(code, 10)
                 makemkvcon_logger.log(loglevel, "%s (%s)", message, code)
 
-                if loglevel == logging.CRITICAL:
+                if loglevel == logging.CRITICAL and p !=None:
```

Comment:
To check if a value is None, `is` / `is not` should be used (see [PEP 8](https://peps.python.org/pep-0008/#programming-recommendations)), so this should be `if loglevel == logging.CRITICAL and self.process is not None:`.